### DISCUSSION TITLE
had an issue, didnt understand, got banned from discord

### DISCUSSION THREAD

**Discussion by arial-improved** [EssentialsX/Essentials#5296] [General]
all i remember is someone telling me to delete my placeholders from the chat format and then i said i have the same issue, i was told to send another essentials dump, did so, they said "check again" i repond with "?" as that isnt very informative and i dont understand what the issue is to start with, i am instantly banned. why was i banned for not understanding an issue that when i asked my server provider said that it was an issue with the plugin and to ask the developers.

my discord is arial#7062 if you want to check

**Comment by xSepticSidx** [comment]
I too just got banned from the discord.
I asked a simple question about kits in the ESS csomething2 channel, I gave a link to pastebin and then was asked to provide a dump so I did that and got immediately banned?

There appears to be no other way to get in touch with the admin of the discord server.
What's going on?
